src/modules/leaderboard.js

```diff
@@ -0,0 +1,27 @@
+import { getData, postData } from './api.js';
+
+const generateLeaderBoard = async (scoreList) => {
+  scoreList.innerHTML = '';
+  const response = await getData();
+  let generateHtml = '';
```

Comment:
- [ OPTIONAL] It would be nice to sort the scores from the highest to the lowest as in a real-world scenario and render the sorted score and their names. 👍